llvm/lib/CodeGen/SelectionDAG/LegalizeDAG.cpp

```diff
@@ -977,6 +977,18 @@ void SelectionDAGLegalize::LegalizeOp(SDNode *Node) {
   TargetLowering::LegalizeAction Action = TargetLowering::Legal;
   bool SimpleFinishLegalizing = true;
   switch (Node->getOpcode()) {
+  // FixMe: If the node represents a poison value, replace it with an undef
+  // value.
```

Comment:
Can you explain what is to be fixed here? The text seems to explain what the code is doing (not what it should do differently).